Removed version blocks (#2493)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/python-models.md

> - Weigh in on [next steps for Python models, beyond v1.3](https://github.com/dbt-labs/dbt-core/discussions/5742).

Reviewer: sugg: remove lines 5 - 9 above

reason: unnecessary since docusaurus automatically builds a right sidebar (page toc) for all H2s and H3s

> - Weigh in on [next steps for Python models, beyond v1.3](https://github.com/dbt-labs/dbt-core/discussions/5742).

Reviewer: sugg for line 11 above to something similar to:

"You can use Python to build your models. "

Reviewer: In the Overview section down below around line 83 talks about prerequisites, sugg grouping prereq there

> - Weigh in on [next steps for Python models, beyond v1.3](https://github.com/dbt-labs/dbt-core/discussions/5742).

Reviewer: ```suggestion
- Share your thoughts and ideas on [next steps for Python models](https://github.com/dbt-labs/dbt-core/discussions/5742).
```

Reviewer: sugg:

- "weigh in" could be colloquial so more challenging to translate 
- remove "1.3" for more future proofing / easier docs maintenance

- Join the **#beta-feedback-python-models** channel in the [dbt Community Slack](https://www.getdbt.com/community/join-the-community/).

> In the following article, you'll see a section titled "❓ **dbt questions**." We are excited to release the first set of functionality in v1.3, which will solve real use cases. We also know this is the first step toward a much wider field of possibility. We don't pretend to have all the answers.

Reviewer: ```suggestion

```

Reviewer: sugg: 

- appropriate for beta but remove now that feature is GA
- couldn't find where section "❓ **dbt questions** " was

Author: The ❓sections are down towards the end of the article

Reviewer: ah, i see the callouts now! i was looking for sections like headings

> We're excited to keep developing our opinionated recommendations and next steps for product development, and we want your help. Comment in the GitHub discussions; leave thoughts in Slack; bring up dbt + Python in casual conversation with colleagues and friends.

Reviewer: ```suggestion

```

Reviewer: sugg remove now that feature is GA. alternatively, use this similar verbiage in a callout asset toward the bottom of this page with links.

## Overview

> dbt Python ("dbt-py") models will help you solve use cases that can't be solved with SQL. You can perform analyses using tools available in the open-source Python ecosystem, including state-of-the-art packages for data science and statistics. Before, you would have needed separate infrastructure and orchestration to run Python transformations in production. Python transformations defined in dbt are models in your project with all the same capabilities around testing, documentation, and lineage.

Reviewer: ```suggestion
dbt Python (`dbt-py`) models can help you solve use cases that can't be solved with SQL. You can perform analyses using tools available in the open-source Python ecosystem, including state-of-the-art packages for data science and statistics. Before, you would have needed separate infrastructure and orchestration to run Python transformations in production. Python transformations defined in dbt are models in your project with all the same capabilities around testing, documentation, and lineage.
```

Python models are supported in dbt Core 1.3 and higher.  Learn more about [upgrading your version in dbt Cloud](https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions) and [upgrading dbt Core versions](https://docs.getdbt.com/docs/core-versions#upgrading-to-new-patch-versions).

To read more about Python models, change the [docs version to 1.3](/docs/build/python-models?version=1.3) (or higher) in the menu bar.

<File name='models/my_python_model.py'>

> 

Reviewer: around line 83 down below, sugg a paragraph that talk about prerequisites with a list with links similar to:

- You need dbt Core version 1.3 or above. For details, see [upgrading your version in dbt Cloud](https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions) and [upgrading dbt Core versions](https://docs.getdbt.com/docs/core-versions#upgrading-to-new-patch-versions)
- Be using a [data platform that support](#supported-data-platforms) `dbt-py` models 
- etc...
